src/main/java/org/openrewrite/java/testing/junit5/ExpectedExceptionToAssertThrows.java

```diff
@@ -32,7 +33,19 @@
 import java.util.stream.Collectors;
 
 /**
- * Replace usages of JUnit 4's @Rule ExpectedException with JUnit 5 Assertions.assertThrows
+ * Replace usages of JUnit 4's @Rule ExpectedException with JUnit 5 Assertions.
+ *
+ * The recipe refactors of ExpectedException to org.junit.jupiter.api.assertThrows,
+ * and org.junit.jupiter.api.assertThrows when applicable.
```

Comment:
This sentence is a bit confusing and repeats `org.junit.jupiter.api.assertThrows` twice. Maybe just delete this sentence?

Reply:
Removed